includes/components/class-admin.php

```diff
@@ -81,7 +84,7 @@ public function __construct( $args = [] ) {
 			add_action( 'admin_init', [ $this, 'check_access' ] );
 
 			// Enqueue scripts.
-			add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_scripts' ] );
+			add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_scripts' ], 11 );
```

Comment:
why this was required?

Reply:
There was a problem that component assets also has admin_enqueue_scripts hook and therefore backend js connects after the pointers